commitlint/commitlint

```diff
@@ -237,6 +237,14 @@ function parse_footer() {
         if (match(tok, CHANGELOG)) {
             CHANGELOG_VAL=tok
             verify_changelog(tok)
+            # Consume the entire paragraph for the 'Changelog' if it is multiline
+            if (next_tok_type == "LINE") {
+                while (tok != EOF && next_tok_type == "LINE") {
+                    advance()
+                }
+                # Require empty line afterwards in paragraph
+                empty_line("A 'Changelog' over multiple lines needs to have a trailing empty line")
```

Comment:
This won't barf if the changelog paragraph is the last line, will it?

Edit: Actually, I guess it will never happen, because signoff always comes after.

Reply:
Exactly :)